src/Payments/Api.php

```diff
@@ -135,6 +135,15 @@ public function process_legacy_payment( PaymentContext $context, PaymentResult &
 			return;
 		}
 
+		// if we have savedPaymentMethodToken in the request, then convert into
+		// the format expected by the post.
+		// phpcs:ignore WordPress.Security.NonceVerification
```

Comment:
Would be good to have  a comment explaining the `phpcs:ignore`. I'm guessing this is because we already trust everything in `POST` because the nonce is checked earlier in the API request handling?